### DISCUSSION TITLE
User guide organisation

### DISCUSSION THREAD

**Discussion by mariomadproductions** [sanni/cartreader#780] [Ideas & Suggestions]
It seems like there are multiple user guide pages, but none of them are really linked anywhere and they seem to cvoer different hardware revisions, its not clear (not to a new user like me anyway).

<https://github.com/sanni/cartreader/wiki/Setup-and-user-guide> <https://github.com/sanni/cartreader/wiki/User-Guide>
<https://github.com/sanni/cartreader/wiki/User-Manual>

I think it'd be a good idea to merge them and link it in the wiki index page, esp. as the guides contain some important safety warnings.

**Comment by Ancyker** [comment]
> https://github.com/sanni/cartreader/wiki/Setup-and-user-guide

is linked on: https://github.com/sanni/cartreader/wiki/Cart-Reader-V2

> https://github.com/sanni/cartreader/wiki/User-Guide

is linked on: https://github.com/sanni/cartreader/wiki/Cart-Reader-V3

> https://github.com/sanni/cartreader/wiki/User-Manual

is linked on: https://github.com/sanni/cartreader/wiki/Cart-Reader-V4

**Reply by mariomadproductions** [reply]
Ah I see. I didn't look very carefully apparently. I guess the pages should just be renamed and one should be made for V5? Also, It seems like the V4 page doesn't have the warnings.

**Reply by Ancyker** [reply]
HW5 is the current version so all/most navbar links are for HW5 unless otherwise noted.

**Reply by mariomadproductions** [reply]
Ah. Is there a user guide for V5? Should I copy the relevant warnings to the V4 page?

**Reply by Ancyker** [reply]
Clicking the system on the nav bar is the guide. I'm not sure what warnings you mean. Also, as far as I know, HW4, unlike HW3, is considered deprecated and not really recommended to build. It doesn't support as many systems as HW5 due to lack of a 5V/3.3V switch.

**Reply by mariomadproductions** [reply]
Ah I mean about using the correct voltages, not plugging in more than one cart at a time, and not plugging/unplugging carts while its on.

**Comment by mariomadproductions** [comment]
@sanni would it be okay if I renamed the pages mentioned above to have the hardware versions in their names (making sure to leave redirects)?

also, where do you think would be a good place to put warnings about using the correct voltages, not plugging in more than one cart at a time, and not plugging/unplugging carts while its on, for the current hardware version?

**Reply by sanni** [reply]
You can edit it however you see fit, that's why I opened up the Wiki for everyone. 👍
